definitions/npm/zxcvbn_v4.x.x/flow_v0.25.x-/zxcvbn_v4.x.x.js

```diff
@@ -0,0 +1,204 @@
+     */
+    online_no_throttling_10_per_second: string | number,
+
+    /**
+     * offline attack. assumes multiple attackers,
+     * proper user-unique salting, and a slow hash function
+     * w/ moderate work factor, such as bcrypt, scrypt, PBKDF2.
+     */
+    offline_slow_hashing_1e4_per_second: string | number,
+
+    /**
+     * offline attack with user-unique salting but a fast hash
+     * function like SHA-1, SHA-256 or MD5. A wide range of
+     * reasonable numbers anywhere from one billion - one trillion
+     * guesses per second, depending on number of cores and machines.
+     * ballparking at 10B/sec.
+     */
+    offline_fast_hashing_1e10_per_second: string | number,
+  };
+
+  declare export type Feedback = {
+    /**
+     * explains what's wrong, eg. 'this is a top-10 common password'.
+     * not always set -- sometimes an empty string
+     */
+    warning: string,
+
+    /**
+     * a possibly-empty list of suggestions to help choose a less
+     * guessable password. eg. 'Add another word or two'
+     */
+    suggestions: string[],
+  };
+
+  declare export type Sequence = {
+    /**
+     * if sequence is ascending.
+     */
+    ascending: boolean,
+
+    /**
+     * the base number of guesses.
+     */
+    base_guesses: number,
+
+    /**
+     * the base match to relate to.
+     */
+    base_matches: string,
+
+    /**
+     * the base token to relate to.
+     */
+    base_token: string,
+
+    /**
+     * the dictionary this sequence was found in.
+     */
+    dictionary_name: string,
+
+    /**
+     * estimated guesses needed to crack sequence.
+     */
+    guesses: number,
+
+    /**
+     * order of magnitude of guesses.
+     */
+    guesses_log10: number,
+
+    /**
+     * sequence index.
+     */
+    i: number,
+
+    /**
+     * sequence section number.
+     */
+    j: number,
+
+    /**
+     * is part of l33t speak.
+     */
+    l33t: boolean,
+
+    /**
+     * how many variations of l33t speak.
+     */
+    l33t_variations: number,
+
+    /**
+     * the word that was matched in dictionary.
+     */
+    matched_word: string,
+
+    /**
+     * what type of pattern the sequence contains.
+     */
+    pattern: string,
+
+    /**
+     * the rank of the sequence in the dictionary.
+     */
+    rank: number,
+
+    /**
+     * how many times the sequence is repeated.
+     */
+    repeat_count: number,
+
+    /**
+     * is reveresed.
+     */
+    reversed: boolean,
+
+    /**
+     * what type of sequence it is.
+     */
+    sequence_name: string,
+
+    /**
+     * how much space the sequence has left.
+     */
+    sequence_space: number,
+
+    /**
+     * the token the sequence is based off.
+     */
+    token: string,
+
+    /**
+     * uppercase variations of the token.
+     */
+    uppercase_variations: number,
+  };
+
+  declare function zxcvbn(password: string, userInputs?: string[]): Result;
+
+  declare module.exports: typeof zxcvbn;
```

Comment:
This could be cleaner like this: `declare export default function zxcvbn(password: string, userInputs?: string[]): Result;`